cli/command/system/version.go

```diff
@@ -17,7 +17,7 @@ import (
 
 var versionTemplate = `Client:
  Version:      {{.Client.Version}}
- API version:  {{.Client.APIVersion}}
+ API version:  {{if eq .Client.DefaultAPIVersion .Client.ActualAPIVersion}}{{.Client.DefaultAPIVersion}}{{else}}{{.Client.APIVersion}}{{end}}
```

Comment:
We should make it so that the default CLI output shows the format as we have now, but allow `--format` to access the individual parts (also see my other comment)